docs/app/pages/components/components-sections/input-controls/typeahead/typeahead.component.html

```diff
@@ -0,0 +1,191 @@
+<div class="row">
+    <div class="col-md-12 has-feedback">
+        <form class="input-typeahead-form">
+            <input [value]="value"
+                   [(ngModel)]="input"
+                   [ngModelOptions]="{standalone: true}"
+                   placeholder="Enter Text"
+                   class="form-control"
+                   (keydown.enter)="dropdownOpen = true;"
```

Comment:
If we look at the tag input component, enter does not open the typeahead, instead the down arrow does, so perhaps change to use it instead.